src/components/Microsoft.DotNet.UpgradeAssistant/TargetFramework/MyTypeTargetFrameworkSelectorFilter.cs

```diff
@@ -33,6 +33,13 @@ public void Process(ITargetFrameworkSelectorFilterState tfm)
                 throw new ArgumentNullException(nameof(tfm));
             }
 
+            if (tfm.Project.OutputType == ProjectOutputType.Library)
```

Comment:
This is tricky because what if a future filter changes the TFM to net5 for a different reason (maybe it has a dependency that targets net5)? Wouldn't we want net5.0-windows in that case?

If we really want these sorts of projects to target net5.0-windows when they target net5 but netstandard when they don't, we need a way to track that in the TFM filter state. I don't think we use netstandard with { Platform = Windows } yet but we could see if that works.

Though maybe this is highlighting that targeting Windows-only things to Linux is a bit weird. What if someone tries to use this netstandard library on Linux? I'm guessing it will fail.

Reply:
My thought was that the `netstandard2.0` library can be shared between `net48` and `net5.0-windows` for applications that are doing the strangler pattern migration approach.

This isn't the focus of the PR though. I'll revert this and we can tackle this when we get more feedback.